keyboards/mykeyclub/jris65/readme.md

```diff
@@ -0,0 +1,32 @@
+# JRIS65
+
+A 65% keyboard sold my Mykeyclub.
```

Comment:
```suggestion
A 65% keyboard sold by Mykeyclub.
```

Reply:
I'm new to doing pull requests. If I press the "commit suggestion" button does that make the change to the branch that contains my pull request (sizezero:jris65) or does it put it somewhere else?

Reply:
It makes the suggested change in your branch 😊